ui/raidboss/data/06-ew/trial/hydaelyn-ex.txt

```diff
@@ -0,0 +1,204 @@
+# Hydaelyn Extreme
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 21:........:40000010:/ window 100000 jump 0
+
+0.0 "--sync--" sync /Engage!/ window 0,1
+16.0 "Heros's Radiance" sync / 1[56]:[^:]*:Hydaelyn:65C1:/
```

Comment:
Just in case somebody doesn't use a countdown (I know, I know), can you add a starts casting line with a sync that hits 0, e.g. `11.0 "Heros's Radiance" sync / 14:[^:]*:Hydaelyn:65C1:/ window 20,20` (or whatever the correct start time is)? Same thing on the other timeline too?

Reply:
I will add the window, there are also a few melee attacks that could possibly be added as well:
00:00:02.229 | Hydaelyn prepares  attack  on 
00:00:03.033 | Hydaelyn attack 
00:00:05.266 | Hydaelyn prepares  attack  on 
00:00:06.070 | Hydaelyn attack 
00:00:08.298 | Hydaelyn prepares  attack  on 
00:00:09.102 | Hydaelyn attack 
00:00:10.977 | Hydaelyn begins casting  Heros's Radiance on Hydaelyn (4.70 sec) 
00:00:15.968 | Hydaelyn casts  Heros's Radiance
00:00:15.968 | Hydaelyn prepares  Heros's Radiance  on
00:00:16.370 | Hydaelyn prepares  attack  on 
00:00:16.772 | Hydaelyn Heros's Radiance